tests/App/Book.php

```diff
@@ -45,7 +47,13 @@ public function toSearchableArray()
     {
         $searchable = parent::toSearchableArray();
         $searchable['author'] = $this->author->only(['name', 'phone_number']);
+        $searchable['db_id'] = $this->id;
```

Comment:
Let's maybe then remove `ElasticScoutDriverPlus\Tests\App\Model::toSearchableArray()`, so we automatically index `id` field and we use it for sorting instead of `db_id`?

Reply:
Hmm, not sure what you mean.

Reply:
By default Scout indexes the` id` field, it's part of `toSearchableArray`. I've changed this behaviour in `ElasticScoutDriverPlus\Tests\App\Model::toSearchableArray()`. If we delete this method, then the model's id will be indexed again, so we don't need the new `db_id` field and we can simply use the `id` field for sorting.

Reply:
I see will try it out soon

Reply:
pushed.